app/messaging/send-message.js

```diff
@@ -0,0 +1,12 @@
+const ffcMessaging = require('ffc-messaging')
```

Comment:
How about:
const { MessageSender } = require('ffc-messaging')

Reply:
This was briefly (and therefore by no means agreed) mentioned during our small chat around module imports that we should perhaps import whole modules for large/3rd party modules to provide more context when we use it: https://eaflood.atlassian.net/wiki/spaces/SFI/pages/4266885138/Module+Imports#Way-to-import

Whatcha think?

Reply:
With reference to the link you provided,  I considered `const { MessageSender } = require('ffc-messaging') `to be more appropriate as we only need 1 function within the module. Also, it is the style/pattern that we adopted in most part of the code files.

Reply:
Reverted